scripts/repo/Sources/Tests/Decrypt.swift

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import ArgumentParser
+import Foundation
+import Logging
+
+extension Tests {
+  /// Command for decrypting the secret files needed for a test run.
+  struct Decrypt: ParsableCommand {
+    nonisolated(unsafe) static var configuration = CommandConfiguration(
+      abstract: "Decrypt the secret files for a test run.",
+      usage: """
+        tests decrypt [--json] [--overwrite] [<json-file>]
```

Comment:
Probably worth mentioning the name of the env var for the `--secret`-less use case.